src/main/java/servlets/FollowListServlet.java

```diff
@@ -0,0 +1,83 @@
+package servlets;
+
+import data.media.PostDAO;
+import data.user.UserDAO;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.annotation.WebServlet;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import model.User;
+import model.media.Post;
+
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.List;
+
+@WebServlet(name = "FollowListServlet", value = "/followList")
+public class FollowListServlet extends HttpServlet {
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        String userIdStr = request.getParameter("userId");
+        String type = request.getParameter("type");
+
+        if (userIdStr == null || type == null || userIdStr.isEmpty() || type.isEmpty()) {
+            response.sendRedirect("home.jsp");
```

Comment:
same thing here about .jsp files and also I think, we can redirect it back to publicProfile.jsp